addons/website/views/snippets/s_social_media.xml

```diff
@@ -29,9 +29,9 @@
     <xpath expr="." position="inside">
         <div data-js="SocialMedia" data-selector=".s_social_media">
             <we-list string="Social Networks" data-add-item-title="Add New Social Network"
-                data-render-list-items="" data-has-default="multiple"
+                data-render-list-items="" data-has-default="multiple" data-name="social_media_list"
                 data-default-value="https://www.example.com" data-id-mode="name"
-                data-generate-id-for-new-items= "true" data-new-elements-toggled="true"/>
+                data-new-elements-not-toggleable="true" data-no-preview="true"/>
```

Comment:
Ok I said I'll allow to remove the old parameter it in stable but actually, I can't allow this one :grimacing: As your new JS code relies on the fact that the XML code is updated... I'll change.